modules/openy_gc_auth/js/auth.behaviors.js

```diff
@@ -0,0 +1,14 @@
+(function ($, Drupal) {
+ Drupal.behaviors.openy_gc_auth_store_hash = {
+   attach: function (context) {
+     // Only run this script on full documents, not ajax requests.
+     if (context !== document) {
+       return;
+     }
+     if (!window.location.hash || window.location.hash === '#') {
+       return;
+     }
+     $.cookie('Drupal_visitor_gc_auth_destination', window.location.hash);
```

Comment:
Let's rename `Drupal_visitor_gc_auth_destination` to `openy_gc_auth_destination`